sync/src/synchronizer/mod.rs

```diff
@@ -707,7 +713,7 @@ mod tests {
         parent = blocks[150].header().to_owned();
         let fork = parent.number();
         for i in 1..=block_number {
-            let parent_epoch = shared2.get_block_epoch(&parent.hash()).unwrap();
+            let parent_epoch = shared2.store().get_block_epoch(&parent.hash()).unwrap();
```

Comment:
Please use `shared.get_block_epoch()`, there is a bit of nuance.

Reply:
don't use ChainProvider::get_block_epoch, it will be deleted later